.github/auto-merge.yml

```diff
@@ -90,14 +82,6 @@
     dependency_name: Particular.PlatformSample.ServicePulse
     update_type: all
 
-- match:
-    dependency_name: PublicApiGenerator
-    update_type: all
-
-- match:
-    dependency_name: SimpleJson
-    update_type: all
-
```

Comment:
We have no references to this package. The source is copied into NServiceBus and committed.

Reply:
It is referenced in core: https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/NServiceBus.Core.csproj#L34

Reply:
My automation was excluding the NServiceBus repo because it was originally designed to look for references to NServiceBus 🤦 

But, anyway, you're correct, so I'll revert this one.